runtime/doc/news.txt

```diff
@@ -210,6 +210,9 @@ These existing features changed their behavior.
   more emoji characters than before, including those encoded with multiple
   emoji codepoints combined with ZWJ (zero width joiner) codepoints.
 
+• |vim.tbl_deep_extend()| no longer ignores any values for which |vim.isarray()|
+  returns `true`.
+
```

Comment:
Should this be moved to the BREAKING CHANGES section?

Reply:
I think this just exposed bugs in both of those plugins.

Reply:
Yes, in a way